OpenRA.Game/Player.cs

```diff
@@ -76,7 +76,7 @@ struct StanceColors
 		// Each player is identified with a unique bit in the set
 		// Cache masks for the player's index and ally/enemy player indices for performance.
 		public LongBitSet<PlayerBitMask> PlayerMask;
-		public LongBitSet<PlayerBitMask> AllyMask = default(LongBitSet<PlayerBitMask>);
+		public LongBitSet<PlayerBitMask> AlliedPlayerMask = default(LongBitSet<PlayerBitMask>);
 		public LongBitSet<PlayerBitMask> EnemyMask = default(LongBitSet<PlayerBitMask>);
```

Comment:
This will need to be updated to `EnemyPlayersMask` (can can we please also have `AlliedPlayersMask` and `AllPlayersMask` with `s`es?)